students/hyska_monika/lesson_06_dicts_tuples_sets_args_kwargs/ArgsInspector.py

```diff
@@ -0,0 +1,13 @@
+# Function called inspect_args that prints passed args
+# and kwargs in human-readable format.
+
+
+def inspect_args(*args, **kwargs):
+    print('Arguments:', ', '.join(map(str, args)))
+    print('Keyword :')
+    for k, v in kwargs.items():
+        print(k, "=", v)
+
+
+my_dict = {'o': '8', 'p': 9}
+inspect_args(5, 7, 8, **my_dict, n=9)
```

Comment:
That will crash with `InvalidSyntax`, `**keyword_args` must go after all named arguments: `inspect_args(5, 7, 8,  n=9, **my_dict)`

Reply:
We discussed this and this was changed in Python3 and is now accepted by the compiler.